### PR TITLE
adding ES parameters to test_pod_reattachtime.py test 

### PR DESCRIPTION
**Pull request by ypersky1980**
Signed-off-by: Yulia Persky <ypersky@redhat.com>

### CODE REVIEW

File: tests/e2e/performance/csi_tests/test_pod_reattachtime.py [hunk 1]
```diff
@@ -232,12 +233,17 @@ def test_pod_reattach_time_performance(
             rsh_cmd = f"exec {pod_name} -- df -h {pod_path}"
             data_written = _ocp.exec_oc_cmd(rsh_cmd).split()[-4]
             logging.info(f"The amount of written data is {data_written}")
+            data_written_str = _ocp.exec_oc_cmd(rsh_cmd).split()[-4]
```
Comment (Avilir) [inline tests/e2e/performance/csi_tests/test_pod_reattachtime.py:236]: why you run it twice ? lines 234 and 236 give the same results

Comment (ypersky1980) [reply tests/e2e/performance/csi_tests/test_pod_reattachtime.py:236]: @Avilir fixed it. thank you for noticing!

**Review by Avilir — DISMISSED** [review]
LGTM

**Review by ramkiperiy — DISMISSED** [review]

**Review by ramkiperiy — APPROVED** [review]
LGTM

**Review by Avilir — APPROVED** [review]
LGTM

**Pull request merged**